packages/demo/webpack.config.js

```diff
@@ -44,7 +44,10 @@ module.exports = {
         new webpack.ContextReplacementPlugin(/moment[/\\]locale$/, /en-ca/),
         new ForkTsCheckerWebpackPlugin({
             typescript: {
-                build: true,
+                // build: true,

```

Comment:
Remove comment?

Reply:
Also did you try installing the `@types/node` package instead of removing the `build: true` option here?

Reply:
Removed.
Yes I tried, but it does not work (I get the same errors)